src/components/AccountDrawer/MiniPortfolio/constants.tsx

```diff
@@ -139,6 +139,35 @@ const TransactionTitleTable: { [key in TransactionType]: { [state in Transaction
   },
 }
 
+export const CancelledTransactionTitleTable: { [key in TransactionType]: string } = {
+  [TransactionType.SWAP]: t`Swap cancelled`,
+  [TransactionType.WRAP]: t`Wrap cancelled`,
+  [TransactionType.ADD_LIQUIDITY_V3_POOL]: t`Add liquidity cancelled`,
+  [TransactionType.REMOVE_LIQUIDITY_V3]: t`Remove liquidity cancelled`,
+  [TransactionType.CREATE_V3_POOL]: t`Create pool cancelled`,
+  [TransactionType.COLLECT_FEES]: t`Collect fees cancelled`,
+  [TransactionType.APPROVAL]: t`Approval cancelled`,
+  [TransactionType.CLAIM]: t`Claim cancelled`,
+  [TransactionType.BUY]: t`Buy cancelled`,
+  [TransactionType.SEND]: t`Send cancelled`,
+  [TransactionType.RECEIVE]: t`Receive cancelled`,
+  [TransactionType.MINT]: t`Mint cancelled`,
+  [TransactionType.BURN]: t`Burn cancelled`,
+  [TransactionType.VOTE]: t`Vote cancelled`,
+  [TransactionType.QUEUE]: t`Queue cancelled`,
+  [TransactionType.EXECUTE]: t`Execute cancelled`,
+  [TransactionType.BORROW]: t`Borrow cancelled`,
+  [TransactionType.REPAY]: t`Repay cancelled`,
+  [TransactionType.DEPLOY]: t`Deploy cancelled`,
+  [TransactionType.CANCEL]: t`Cancellation cancelled`,
```

Comment:
Ooof this phrasing blows

Reply:
Agreed 🤣 but nothing in our app will currently produce this state

Reply:
what kind of cancellation is this? Can we add some more context to this string?